test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -175,21 +187,91 @@ class db_size_statistic : public statistic {
         file_names.push_back(std::string(DEFAULT_DIR) + "/" + WT_HS_FILE);
         file_names.push_back(std::string(DEFAULT_DIR) + "/" + WT_METAFILE);
 
-        return file_names;
+        return (file_names);
     }
 
     database &_database;
     int64_t _limit;
 };
 
+class postrun_statistic_check {
```

Comment:
What's the rationale for not extending statistic?

Reply:
How would it be helpful?

The only reason I would inherit from something is to get polymorphic behavior. I'm not using this as a regular stat in the `runtime_monitor` so I don't see a reason to do that.

Reply:
Fair enough.

Reply:
Although it is misleading to have a base class statistic extended by some but not all, could we rename statistic to runtime_statistic?

Reply:
a0ff4c2